src/spyglass/spikesorting/v1/curation.py

```diff
@@ -0,0 +1,504 @@
+from typing import Dict, List, Union
+
+import datajoint as dj
+import numpy as np
+import pynwb
+import spikeinterface as si
+import spikeinterface.curation as sc
+import spikeinterface.extractors as se
+
+from spyglass.common.common_ephys import Raw
+from spyglass.common.common_nwbfile import AnalysisNwbfile
+from spyglass.spikesorting.v1.recording import SpikeSortingRecording
+from spyglass.spikesorting.v1.sorting import SpikeSorting, SpikeSortingSelection
+
+schema = dj.schema("spikesorting_v1_curation")
+
+valid_labels = ["reject", "noise", "artifact", "mua", "accept"]
+
+
+@schema
+class CurationV1(dj.Manual):
+    definition = """
+    # Curation of a SpikeSorting. Use `insert_curation` to insert rows.
+    -> SpikeSorting
+    curation_id=0: int
+    ---
+    parent_curation_id=-1: int
+    -> AnalysisNwbfile
+    object_id: varchar(72)
+    merges_applied: enum("True", "False")
```

Comment:
Sorry - this just occurred to me. It will accept python bools and represent them as 1/0

```suggestion
    merges_applied: bool
```